README.md

```
@@ -1,7 +1,26 @@
 # CAI 2840C
 
+
+How to install needed packages
+
+```
+pip install -r requirements.txt
+```
+
+Alternative method
+
 ```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)